### PR TITLE
Added a Makefile and a rdef file

### PR DESCRIPTION
**Pull request by picapi**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+#		you can simply specify XXX for the library. (e.g. the entry for
+#		"libtracker.so" would be "tracker")
+#
+#	-	for GCC-independent linking of standard C++ libraries, you can use
+#		$(STDCPPLIBS) instead of the raw "stdc++[.r4] [supc++]" library names.
+#
+#	- 	if your library does not follow the standard library naming scheme,
+#		you need to specify the path to the library and it's name.
+#		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
+LIBS =  /boot/system/lib/libroot.so /boot/system/lib/libbe.so /boot/system/lib/libsupc++.so
```
Comment (korli) [inline Makefile]: be supc++

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+#	or libXXX.a naming scheme. You can specify full paths or paths relative
+#	to the Makefile. The paths included are not parsed recursively, so
+#	include all of the paths where libraries must be found. Directories where
+#	source files were specified are	automatically included.
+LIBPATHS = 
+
+#	Additional paths to look for system headers. These use the form
+#	"#include <header>". Directories that contain the files in SRCS are
+#	NOT auto-included here.
+SYSTEM_INCLUDE_PATHS =  /boot/develop/headers/be /boot/develop/headers/cpp /boot/develop/headers/posix /boot/home/config/include
```
Comment (korli) [inline Makefile]: This should work also when empty.

**Comment by picapi** [comment]
Thanks for letting me know about the unneeded imports, I've fixed them now!

**Comment by picapi** [comment]
I've applied my patches to the branch, which should result in a functional program, and at least do so on my end.

**Pull request merged**